src/main/kotlin/no/nav/pensjon/kalkulator/vedtak/LoependeVedtak.kt

```diff
@@ -0,0 +1,16 @@
+package no.nav.pensjon.kalkulator.vedtak
+
+import java.time.LocalDate
+
+data class LoependeVedtak(
+    val alderspensjon: LoependeVedtakDetaljer?,
+    val ufoeretrygd: LoependeVedtakDetaljer?,
+    val afpPrivat: LoependeVedtakDetaljer?,
+    val afpOffentlig: LoependeVedtakDetaljer?,
```

Comment:
Burde denne hete f.eks. `afpOffentligLivsvarig` for å gjøre det klarere hva som skiller denne fra `afpOffentligForBrukereFoedtFoer1963`?

Reply:
Tankegangen var at `afpOffentlig` er et godt etablert begrep i teamet og i pensjonskalkulator-koden, så det kan være lurt å la det nye få et lengre navn som viser tydelig hvilke brukere det gjelder.

Jeg tenker, hvis vi ville ha `afpOffentligLivsvarig` og `afpOffentlig`, så burde vi bruke `afpOffentligLivsvarig` i alle andre steder for å unngå fremtidig forvirring.
Hva tenker du?

Reply:
Vi kan la det være `afpOffentlig`